src/utils/address.ts

```diff
@@ -11,6 +11,12 @@ import { assertInRange, toHex } from './num';
  * Format a hex number to '0x' and 64 characters, adding leading zeros if necessary.
  * @param {BigNumberish} address
  * @returns {string} Hex string : 0x followed by 64 characters. No upper case characters in the response.
+ * @example
+ * ```typescript
+ * const address = 0x7ee790591d9fa3efc87067d95a643f8455e0b8190eb8cb7bfd39e4fb7571fdf;
+ * const result = addAddressPadding(address);
+ * // result = "0x07ee790591d9fa3efc87067d95a643f8455e0b8190eb8cb7bfd39e4fb7571fdf"
```

Comment:
Address is a string. To use between quotes.
You should use an address that needs more zeros (for example 3), as we have a function that optionally add a zero to have an odd number of characters.